Dockerfile

```
@@ -9,4 +9,5 @@ RUN yes | yum update \
  && yes | yum localinstall *.rpm \
  && rm java-1.8.0-amazon-corretto-1.8.0_192.b12-1.amzn2.x86_64.rpm \
        java-1.8.0-amazon-corretto-devel-1.8.0_192.b12-1.amzn2.x86_64.rpm
```

Don't forget the trailing backslash on the previous line! Without that, the build fails with: `Error response from daemon: Dockerfile parse error line 12: unknown instruction: &&`

```suggestion
        java-1.8.0-amazon-corretto-devel-1.8.0_192.b12-1.amzn2.x86_64.rpm \
```